crates/topos/src/components/node/services/process.rs

```diff
@@ -113,29 +111,12 @@ pub(crate) fn spawn_tce_process(
         config.p2p.public_addresses = vec![addr];
     }
 
-    let tce_config = TceConfiguration {
-        is_bootnode: config.p2p.is_bootnode.unwrap_or_default(),
-        boot_peers,
-        validators,
-        auth_key,
-        signing_key: keys.validator.map(AuthKey::PrivateKey),
-        listen_addresses: config.p2p.listen_addresses,
-        public_addresses: config.p2p.public_addresses,
-        tce_params,
-        api_addr: config.grpc_api_addr,
-        graphql_api_addr: config.graphql_api_addr,
-        metrics_api_addr: config.metrics_api_addr,
-        storage: StorageConfiguration::RocksDB(Some(config.db_path)),
-        network_bootstrap_timeout: Duration::from_secs(config.network_bootstrap_timeout),
-        minimum_cluster_size: config
-            .minimum_tce_cluster_size
-            .unwrap_or(NetworkConfig::MINIMUM_CLUSTER_SIZE),
-        version: env!("TOPOS_VERSION"),
-    };
+    config.version = env!("TOPOS_VERSION");
```

Comment:
How is this ENV var set? What value does `config.version` take if it's not set? We use figment elsewhere to define a hierarchy of config value sources. Can we do the same for the version? Ideally I'd want this to use the crate version of the `topos` crate + the short hash of the build, and not have a user-inputted value at all.

Reply:
the version is computed in the `build.rs`